pcsx2-qt/EmuThread.cpp

```diff
@@ -889,4 +890,20 @@ DEFINE_HOTKEY("ToggleFullscreen", "General", "Toggle Fullscreen", [](bool presse
 	if (!pressed)
 		g_emu_thread->toggleFullscreen();
 })
+// Input Recording Hot Keys
+DEFINE_HOTKEY("InputRecToggleMode", "Input Recording", "Toggle Recording Mode", [](bool pressed) {
+	if (!pressed) // ?? - not pressed so it is on key up?
+	{
+		g_InputRecordingControls.RecordModeToggle();
+	}
+})
+// TODO - Vaser - the way the pause/resuming used to work is broken on Qt, needs a rewrite. 
+//	- Currently if you frame advance you can't get out of frame advancing!
+DEFINE_HOTKEY("InputRecFrameAdvance", "Input Recording", "Frame Advance", [](bool pressed) {
+	if (!pressed) // ?? - not pressed so it is on key up?
+	{
+		g_InputRecordingControls.FrameAdvance();
+		g_InputRecordingControls.ResumeCoreThreadIfStarted();
```

Comment:
There shouldn't be any mention of "core thread" in Qt code, it doesn't exist.

Reply:
will be cleaned up when the class is cleaned up, the naming has not been touched yet.